quickview/ui_mock.py

```diff
@@ -0,0 +1,55 @@
+from getter_ui import Ui_Dialog
+from PyQt4 import QtGui
+
+
+class Getter(Ui_Dialog,QtGui.QDialog):
+    def __init__(self,title):
+        super(QtGui.QDialog,self).__init__()
+        self.setupUi(self)
+        self.setWindowTitle(title)
+        self.btnInt.clicked.connect(self.on_click)
+        self.btnFloat.clicked.connect(self.on_click)
+        self.btnFilepath.clicked.connect(self.on_click)
+        self.btnBool.clicked.connect(self.on_click)
+        self.btnString.clicked.connect(self.on_click)
+        self.lineEdit.returnPressed.connect(self.on_click)
+        self._done = False
+        self.exec_()
+
+    def is_done(self):
+        return self._done
+
+    def get_data(self):
+        return self._data
+
+    def on_click(self):
+        sender = self.sender()
+        text = str(self.lineEdit.text())
+        if sender == self.btnInt:
```

Comment:
Not sure what's going on here. You have different buttons for different types of input?